### ISSUE TITLE
`init.el` module help doesn't seem to work

### ISSUE THREAD

**Issue by nilp0inter** [nix-community/nix-doom-emacs#434]
Hi, I am fairly new to `doom-emacs` and `nix-doom-emacs`.  To learn about all doom-emacs functionality a fairly common workflow would be:

1. Open `init.el` file via `SPC f p`
2. Put the cursor over a module (docker for instance)
3. Press `K` to display the module help

Step 3 doesn't work at all for me, event with the minimal `flake.nix`.

Is this expected behaviour? Any workaround or method to learn about the functionality of `doom-emacs` without leaving `nix-doom-emacs?

Thank you very much! I love the project 😀

**Comment by ckiee** [comment]
Yeah, this has been broken for a while. See #353 and #316 — we haven't been able to update our Doom version, good luck!

**Comment by nilp0inter** [comment]
Thank you for the fast response, @ckiee !

I don't mind using an older version of `doom-emacs` in the meanwhile, since I want to learn to use it first :)

Could you point me to a commit when this functionality worked?

Thanks!

**Comment by ckiee** [comment]
I don't know of one, it was too long ago. hlissner (the upstream maintainer) runs his Doom on NixOS without much of anything special: you can do that too if you want- there's also a `shell.nix` in there you could try.

(Maybe you'll even learn enough Elisp to try to revive this accursed project)

**Comment by nilp0inter** [comment]
I appreciate your insight. For the time being, I am gonna stick with running `doom-emacs` as hlissner does.  I'll keep an eye on this repo, though, in case the situation improves 🤞.

Thank you!
